PageObjectPattern/src/test/java/serenity/pages/ForgotPasswordPage.java

```diff
@@ -0,0 +1,35 @@
+package serenity.pages;
+
+import org.openqa.selenium.By;
+import net.serenitybdd.core.steps.UIInteractionSteps;
+import net.serenitybdd.screenplay.ensure.BooleanEnsure;
+import net.serenitybdd.screenplay.ensure.Ensure;
+import net.thucydides.core.annotations.Step;
+
+//Elements and Methods on Forgot password page
```

Comment:
I remember I left a comment to your previous PR about comments. You can either remove them from all classes or if you need them, just enclose into /** */ comments block

Reply:
done